ios/Podfile.lock

```diff
@@ -1178,10 +1178,6 @@ PODS:
     - React-Core
   - react-native-launch-arguments (4.0.2):
     - React
-  - react-native-live-markdown (0.1.0):
-    - glog
-    - RCT-Folly (= 2022.05.16.00)
-    - React-Core
```

Comment:
this was accidentally removed?

Reply:
It's not removed, but renamed, you can see the package below. I suppose that in the version bump PR the `pod install` command was not executed, so `Podfile.lock` was not edited

Reply:
I run `pod install` on latest main and diff doesn't occur

Reply:
Ok, you're right, I know what happened. `npm i` didn't update `react-native-live-markdown.podspec` file without deleting `node_modules`

Reply:
I'll push the change in a moment